pages/learn/welcome/concepts.md

```diff
@@ -32,3 +32,7 @@ A versioned build of the host OS, specific to a fleet and device type.
 ### Base image
 
 A specific combination of a Linux distribution and additional packages, used as the base operating environment for a service.
+
+### Cloudlink
+
+Previously called {{ $names.company.lower }}VPN, Cloudlink is the connection that allows {{ $names.company.lower }} to have to have a secure tunnel to a device. This uses openVPN as an underlying technology to provide features like ssh access, the public device URL and the web terminal.
```

Comment:
```suggestion
Previously called {{ $names.company.lower }}VPN, this is the connection that allows {{ $names.cloud.lower }} to have a secure tunnel to the device. This feature in balenaCloud uses openVPN as an underlying technology to enable other features like SSH access, the public device URL, and the web terminal.
```

Nitpick: How about we link all 3 features in the end so someone reading this for the first time can figure what these are.

Reply:
done, thanks @vipulgupta2048